owtf/api/handlers/auth.py

```diff
@@ -0,0 +1,230 @@
+"""
+owtf.api.handlers.auth
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+"""
+from owtf.api.handlers.base import APIRequestHandler
+from owtf.lib.exceptions import APIError
+from owtf.models.user import User
+from datetime import datetime, timedelta
+import tornado
+import bcrypt
+import json
+import base64
+import uuid
+import jwt
+import re
+
+JWT_SECRET = 'my_secret_key'
```

Comment:
ditto for lines 18-20 - all of this should be in settings

Reply:
Changed.